ONTOLOGY_GUIDE.md

```diff
@@ -297,23 +313,26 @@ These tests are run every time a change is introduced via a pull request in GitH
 ```
           
 ## Candidate Tests for Quality Compliance
-In addition to the tests listed above, the FIBO leadership team is contemplating implementing the following tests (through automation where possible, but in review processes at a minimum), to further ensure the quality of the ontologies we publish.  These include:
+In addition to the tests listed above, the FIBO Community is contemplating implementing the following tests (through automation where possible, but in review processes at a minimum), to further ensure the quality of the ontologies we publish.  These include:
 
 ### T1. Polysemous elements
 
 An ontology element whose name has different meanings is included in the ontology to represent more than one conceptual idea. For example, the class “Theatre” is used to represent both the artistic discipline and the place in which a play is performed.
 
```

Comment:
Polysemous elements should be documented as a policy not as a potentially automatable test (I don't see any way it COULD be automated).